src/main/java/io/github/sornerol/chess/pubapi/domain/game/ArchiveGame.java

```diff
@@ -41,4 +42,25 @@ public class ArchiveGame extends GameBase {
      */
     @JsonProperty("accuracies")
     private Accuracies accuracies;
+    
+    /**
+     * The Tcn
+     */
+    @JsonProperty("tcn")
+    private String tcn;
+
+    /**
+     * The UUID of the Player
+     */
+    //TODO Check if this can be replaced by a UUID
+    @ApiStatus.Experimental
+    @JsonProperty("uuid")
+    private String uuid;
+
+    /**
+     * The Setup
+     */
+    @ApiStatus.Experimental
```

Comment:
Is there a source indicating these fields are experimental?

Reply:
Because i also dont know for what this field stands